roles/openstack-setup/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
   - role: endpoints
+  - role: nova-common
```

Comment:
I have created variable structure and also added openstack-meta role dependency to openstack-setup under PR https://github.com/blueboxgroup/ursula/pull/2844

Please go ahead and define variable in openstack-meta/defaults/main.yml
openstack_meta.nova.force_flavors